sdk/rs/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "squads-multisig"
-version = "2.0.1"
```

Comment:
Can we change this to match the new program version 2.1.0?

Reply:
done with [53e90c4](https://github.com/Squads-Protocol/v4/pull/139/commits/53e90c499dafce3c18cdca6e1f3a7f417d491d48) 🫡